i18n/en.json

```diff
@@ -1815,6 +1815,53 @@
+  "admin.reporting.workspace_optimization.ease_of_management.ldap.description": "You’ve reached over 100 users, we can reduce your manual management pains through AD/LDAP with features like easier onboarding, automatic deactivations and automatic role assignments.",
+  "admin.reporting.workspace_optimization.ease_of_management.ldap.title": "AD/LDAP integration recommended",
+  "admin.reporting.workspace_optimization.ease_of_management.title": "Ease of management",
+  "admin.reporting.workspace_optimization.overall_workspace_score": "Overall Workspace Score",
+  "admin.reporting.workspace_optimization.overall_workspace_score_description": "Stay on top of how optimized your workspace is! This score is a total of the categories of key indicators of health and growth below. Ensure your workspace is in its most optimized state so you and your user base get the most out of Mattermost.",
+  "admin.reporting.workspace_optimization.performance.description": "Your server could use some performance tweaks.",
+  "admin.reporting.workspace_optimization.performance.search.description": "Your server has reached over 500 users and 2 million posts which could result in slow search performance. We recommend starting an enterprise trial with the elastic search feature for better performance.",
+  "admin.reporting.workspace_optimization.performance.search.title": "Search performance",
+  "admin.reporting.workspace_optimization.performance.title": "Performance",
+  "admin.reporting.workspace_optimization.security.description": "There are security concerns you should look at.",
+  "admin.reporting.workspace_optimization.security.login_attempts.description": "{attempts} Failed login attempts have been detected. We recommend looking at the security logs to understand the risk.",
+  "admin.reporting.workspace_optimization.security.login_attempts.title": "Failed login attempts detected",
+  "admin.reporting.workspace_optimization.security.title": "Security Concerns",
+  "admin.reporting.workspace_optimization.session_length.title": "Session length is still set to defaults",
+  "admin.reporting.workspace_optimization.startTrial": "Start Trial",
+  "admin.reporting.workspace_optimization.title": "Workspace Optimization",
+  "admin.reporting.workspace_optimization.updates.description": "You have an update to consider",
+  "admin.reporting.workspace_optimization.updates.server_version.status.error.description": "Placeholder: Mattermost {version} contains a medium level security fix. Upgrading to this release is recommended.",
+  "admin.reporting.workspace_optimization.updates.server_version.status.error.title": "{type} version update available",
+  "admin.reporting.workspace_optimization.updates.server_version.status.ok.description": "Placeholder: Nothing to do here. All good!",
+  "admin.reporting.workspace_optimization.updates.server_version.status.ok.title": "Your Mattermost server is running the latest version",
```

Comment:
```suggestion
  "admin.reporting.workspace_optimization.updates.server_version.status.ok.title": "Your Mattermost server is running the latest version.",
```

Reply:
again @michelengelen @cwarnermm is this the subitem or the accordion header description? if its the header i have "Your workspace is completely up to date!" in the design